rsts/deployment/aws/opta.rst

```diff
@@ -9,14 +9,18 @@ load robustly and securely.
 * The Kubernetes cluster needs to run securely and robustly
 * The sandbox's object store must be replaced with a production-grade storage system
 * The sandbox's PostgreSQL database must be replaced with a production-grade deployment of PostgreSQL
-* All the above have to be done in a secure manner
-* (Optionally) An official DNS domain must be created
-* (Optionally) A production-grade email sending system must be provisioned and configured
+
+All the above have to be done in a secure manner
+
+* (Optional) Create an official DNS domain
+* (Optional) Provision and configure a production-grade email sending system
 
 A Flyte user may provision and orchestrate this setup by themselves, but the Flyte team has partnered with the
 `Opta <https://github.com/run-x/opta>`_ team to create a streamlined production deployment strategy for AWS with
 ready-to-use templates provided in the `Flyte repo <https://github.com/flyteorg/flyte/tree/master/opta/aws>`__.
 
+.. warning:: Opta is no longer actively in development. You can `migrate to terraform  <https://docs.opta.dev/features/terraform/#migrate-from-opta-to-terraform>`__ to generate terraform code from the opta file.
```

Comment:
Hmm, not a fan that our recommended installation path relies on something that's no longer being maintained and has this warning.  It doesn't inspire any confidence. At the very least could we add that we're trying to create and maintain a terraform module and a timeline for when you think we'll have it?

Reply:
We can port all the steps from opta to flyte

Reply:
@SmritiSatyanV can you port the migration docs from opta to flyte?

Reply:
Sure @evalsocket